src/index.html

```diff
@@ -4,15 +4,89 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
```

Comment:
The meta viewport tag has its attributes spread across multiple lines, which exceeds the recommended maximum of 80 characters per line. Consider breaking the attributes into separate lines to improve readability and adhere to the 80 character limit.